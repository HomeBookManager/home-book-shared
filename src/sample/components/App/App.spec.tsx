import { render } from '@testing-library/react';

// components
import App from './App';

describe('AppInitializer snapshots', () => {
  it('should render AppInitializer', () => {
    // before
    const { asFragment } = render(<App />);

    // result
    expect(asFragment()).toMatchSnapshot();
  });
});
