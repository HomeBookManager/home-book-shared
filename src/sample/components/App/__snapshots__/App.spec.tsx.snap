// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppInitializer snapshots should render AppInitializer 1`] = `
<DocumentFragment>
  App
</DocumentFragment>
`;
